pages/docs/user-docs/build-docker-module.md

```diff
@@ -0,0 +1,43 @@
+---
+title: docker bootstrap module
+sidebar: user_docs
+permalink: build-docker-module
+folder: docs
+toc: false
+---
+
+This module allows you to build a container from an existing container hosted on Docker Hub or another Docker registry. 
+
```

Comment:
I would be poignant that it's not a container, but layers. Something like "from image layers hosted on Docker Hub or another Docker registry."